components/experience-section/ExperienceEdit.tsx

```diff
@@ -79,42 +80,47 @@ const ADD_EXPERIENCE = gql`
 interface ExperienceEditProps extends DialogProps {
   t: TFunction;
   id?: string;
-  experience?: Experience;
+  experiences: Experience[];
   talent: Talent;
   onClose: () => void;
 }
 
 export const ExperienceEdit = ({
   talent,
-  id,
-  experience = {
-    id: '',
-    talent,
-    lineOfWork: 'NURSE',
-    employer: {
-      id: '',
-      name: '',
-      address: {
-        city: '',
-        isoCode: '',
-      },
-    },
-    duration: {
-      from: {
-        timeStamp: Date.now().toString(),
-      },
-      to: {
-        timeStamp: '',
-      },
-    },
-    isComplete: false,
-  },
+  id = '',
+  experiences = [],
   t,
   ...props
 }: ExperienceEditProps): React.ReactElement => {
+  const newExperience: Experience = useMemo(
```

Comment:
ah, useMemo. Really cool. I read about it a few weeks ago but never used it.

Reply:
eslint suggested it, so I did. Who am I to doubt eslint.